### ISSUE TITLE
nitric oxide synthase levels

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3397]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. abnormal nitric oxide synthase level
2. decreased nitric oxide synthase 1 level
3. increased nitric oxide synthase 1 level
4. decreased nitric oxide synthase 2 level
5. increased nitric oxide synthase 2 level
6. decreased nitric oxide synthase 3 level
7. increased nitric oxide synthase 3 level
8. abnormal NOS2/NOS3 ratio
9. decreased NOS2/NOS3 ratio
10. increased NOS2/NOS3 ratio

>2. Synonyms (e.g., absent spleen)

nitric oxide synthase - NOS; nitric oxide synthase 1 - nNOS; nitric oxide synthase 2 - iNOS; nitric oxide synthase 3 - eNOS; increased - elevated; decreased - reduced

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. aberrant concentration of the enzyme(s) that catalyzes the synthesis of nitric oxide from L-arginine.
2. decreased concentration of the constitutive neuronal enzyme that catalyzes the synthesis of nitric oxide from L-arginine.
3. increased concentration of the constitutive neuronal enzyme that catalyzes the synthesis of nitric oxide from L-arginine.
4. decreased concentration of the inducible enzyme that catalyzes the synthesis of nitric oxide from L-arginine.
5. increased concentration of the inducible enzyme that catalyzes the synthesis of nitric oxide from L-arginine.
6. decreased concentration of the constitutive endothelial enzyme that catalyzes the synthesis of nitric oxide from L-arginine.
7. increased concentration of the constitutive endothelial enzyme that catalyzes the synthesis of nitric oxide from L-arginine.
8. aberrant ratio of the concentrations of NOS2/NOS3, two of the enzymes that catalyze the synthesis of nitric oxide from L-arginine.
9. decreased ratio of the concentrations of NOS2/NOS3, two of the enzymes that catalyze the synthesis of nitric oxide from L-arginine.
10. increased ratio of the concentrations of NOS2/NOS3, two of the enzymes that catalyze the synthesis of nitric oxide from L-arginine.
 
PMID:32048631 

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal enzyme/coenzyme level (MP:0005319) and abnormal nitric oxide homeostasis (MP:0003957)
2. - 7.  abnormal nitric oxide synthase level
9. - 10.  abnormal NOS2/NOS3 ratio

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0003-3645-6803
